Pod/UI/Classes/UILabel+BonMotUtilities.m

```diff
@@ -0,0 +1,37 @@
+//
+//  UILabel+BonMotUtilities.m
+//  BonMot
+//
+//  Created by Nora Trapp on 3/2/16.
+//
+//
+
+#import "BonMot.h"
+@import ObjectiveC.runtime;
+
+@implementation UILabel (BonMotUtilities)
+
+- (void)setBonTextable:(id<BONTextable>)textable
+{
+    objc_setAssociatedObject(self, @selector(bonTextable), textable, OBJC_ASSOCIATION_COPY_NONATOMIC);
```

Comment:
#nth it occurred to me that `BONTextable` should conform to `NSCopying`, and then `BONChain` and `BONText` don't have to, explicitly. Without doing that, this line doesn't quite make sense, because you're using `OBJC_ASSOCIATION_COPY_NONATOMIC` on something that doesn't explicitly support copying.